### ISSUE TITLE
prove facial expression recognition application with various data

### ISSUE THREAD

**Issue by mkh48v** [omnis-labs-company/omnis#403]
check https://en.wikipedia.org/wiki/Facial_expression_databases

**Comment by 2013-11390** [comment]
I check fer2013 set and it has accuracy more than 0.8 in validation set. I'll try facial expression recognition with face landmarks for more accuracy.

**Comment by 2013-11390** [comment]
RBM (Yichuan Tang) — 71.162%
UNSUPERVISED (Yingbo Zhou & Chetan Ramaiah) — 69.267%
MAXIM MILAKOV (Maxim Milakov) — 68.821%
RADU+MARIUS+CRISTI (Radu Ionescu & Marius Popescu & Cristian Grozea) — 67.484%
They are results of leaders in kaggle competition for same dataset(fer2013)

**Comment by 2013-11390** [comment]
I get 0.714 accuracy in test dataset and get 0.6375 in validation accuracy.
